### ISSUE TITLE
[runner] salt-run cache.clear_mine_func broken, can't take clear_mine_func

### ISSUE THREAD

**Issue by arthurlogilab** [saltstack/salt#27498]
```
root@salt:/srv/salt/states# salt-run -d cache.clear_mine_func
'cache.clear_mine_func:'

    Clear the cached mine function data of the targeted minions

    CLI Example:

        salt-run cache.clear_mine_func tgt='*' clear_mine_func='network.interfaces'
    
root@salt:/srv/salt/states# salt-run cache.clear_mine_func tgt='host.logilab.fr' clear_mine_func='test.ping'
The following keyword arguments are not valid: clear_mine_func=test.ping
```

```
# salt-run --versions-report
                  Salt: 2015.5.3
                Python: 2.7.9 (default, Mar  1 2015, 12:57:24)
                Jinja2: 2.7.3
              M2Crypto: 0.21.1
        msgpack-python: 0.4.2
          msgpack-pure: Not Installed
              pycrypto: 2.6.1
               libnacl: Not Installed
                PyYAML: 3.11
                 ioflo: Not Installed
                 PyZMQ: 14.4.0
                  RAET: Not Installed
                   ZMQ: 4.0.5
                  Mako: 1.0.0
               Tornado: Not Installed
 Debian source package: 2015.5.3+ds-1~bpo8+1
```

**Comment by jfindlay** [comment]
@arthurlogilab, thanks for the report.

**Comment by rallytime** [comment]
@arthurlogilab It looks like the docs are just wrong for that kwarg. The example should read `salt-run cache.clear_mine_func tgt='*' clear_mine_func_flag='network.interfaces'`

I've fixed this in #31191.
